src/libraries/Microsoft.Extensions.DependencyInjection/src/ScopePool.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using Microsoft.Extensions.DependencyInjection.ServiceLookup;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    internal class ScopePool
+    {
+        // Modest number to re-use. We only really care about reuse for short lived scopes
+        private const int s_maxQueueSize = 128;
```

Comment:
(nit) Use PascalCasing for consts. #12 in [https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)